Dockerfile

```diff
@@ -5,11 +5,8 @@ FROM alpine:latest
 
 RUN apk update && apk add go delve build-base gcc libc6-compat
 
-# A workaround for intellij debugging support
```

Comment:
Since this is a dev Dockerfile (we should probably rename it to Dockerfile.dev), can we keep this workaround ? It allows attaching a debugger in my IDE.

Reply:
Sure, done

Reply:
👍